### PR TITLE
fix:package.json updated

### PR DESCRIPTION
**Pull request by sambit-sankalp**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
- package.json file updated

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,26 +1,27 @@
 {
-  "name": "project-oregano",
+  "name": "website-v2",
```
Comment (riteshsp2000) [inline package.json:2]: change this to tedxnitrourkela-website-v2

File: package.json [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
   "private": true,
-  "description": "Official repository for HackNITR 3.0",
+  "description": "Official repository for TEDxNITRourkela",
```
Comment (riteshsp2000) [inline package.json:4]: Official repository for the website of TEDxNITRourkela

File: package.json [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
   "version": "0.1.0",
   "license": "MIT",
```
Comment (riteshsp2000) [inline package.json:6]: I don't think we are using MIT License. Confirm once from @rutajdash regarding this, he knows the specifics.

**Review by riteshsp2000 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (rutajdash) [reply package.json:6]: put "SEE LICENSE IN LICENSE.md"

**Review by riteshsp2000 — APPROVED** [review]

**Pull request merged**